modules/deleting-a-project-using-the-web-console.adoc

```diff
@@ -4,26 +4,39 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="deleting-a-project-using-the-web-console_{context}"]
-= Deleting a project using the web console
+= Deleting a project by using the web console
 
-You can delete a project by using the {product-title} web console.
+You can delete a project by using the web console.
 
-[NOTE]
-====
-If you do not have permissions to delete the project, the *Delete Project*
-option is not available.
-====
+.Prerequisites
 
-.Procedure
+* You have created a project.
+* You have the required permissions to delete the project.
 
-. Navigate to *Home* -> *Projects*.
+.Procedure
 
-. Locate the project that you want to delete from the list of projects.
+** If you are using the *Administrator* perspective:
+.. Navigate to *Home* -> *Projects*.
+.. Select a project from the list.
+.. Click the *Actions* drop-down menu for the project and select *Delete Project*.
++
+[NOTE]
+====
+If you do not have the required permissions to delete the project, the *Delete Project* option is not available.
+====
 
-. On the far right side of the project listing, select *Delete Project* from the
-Options menu {kebab}.
+. In the *Delete Project?* pane, confirm the deletion by entering the name of your project.
+. Click *Delete*.
 
-. When the *Delete Project* pane opens, enter the name of the project that
-you want to delete in the field.
+** If you are using the *Developer* perspective:
+.. Navigate to the *Project* page.
+.. Select the project that you want to delete from the *Project* drop-down menu.
+.. Click the *Actions* drop-down menu for the project and select *Delete Project*.
++
+[NOTE]
+====
+If you do not have the required permissions to delete the project, the *Delete Project* option is not available.
```

Comment:
I feel like this reads better if we rearrange the sentence to remove the comma, personally.

```adoc
The *Delete Project* option is not available if you do not have the required permissions to delete the project.
```